public/app/config/settings.js

```diff
@@ -0,0 +1,41 @@
+(function () {
+    'use strict';
+
+    define(
+        [
+            'util/container'
+        ],
+        function (container) {
+            return function () {
+                container.register('settings.library', {
```

Comment:
These are just better centralised now in their own DI components, one per collection.